examples/discovery-search-app/cypress/integration/pagination/spec.ts

```diff
@@ -0,0 +1,210 @@
+const itemsPerPageOptions = ['5', '10', '20', '30', '40', '50'];
+
+describe('Pagination', () => {
+  beforeEach(() => {
+    cy.server({ force404: true });
+    cy.fixture('basic/query.json').as('queryJSON');
+    cy.fixture('basic/collections.json').as('collectionsJSON');
+    cy.fixture('basic/componentSettings.json').as('componentSettingsJSON');
+    cy.fixture('pagination/multiPageResults.json').as('multiPageResultsJSON');
+    cy.route('GET', '**/component_settings?version=2019-01-01').as('componentSettings');
+    cy.route('POST', '**/query?version=2019-01-01', '@queryJSON').as('searchQuery');
+    cy.route('GET', '**/collections?version=2019-01-01', '@collectionsJSON').as('getCollections');
+    cy.visit('/');
+  });
+
+  describe('When the application loads', () => {
+    it('pagination component is present', () => {
+      cy.get('.bx--pagination').should('exist');
```

Comment:
Doesn't a query have to be submitted before the pagination component will exist?

Reply:
Right now it's present on load of the example app. If it's not supposed to be, then we might need to look into creating an issue to fix it